operations/helm/charts/grafana-agent/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ Unreleased
 
 - Allow users of the helm chart to configure the image registry either at the image level or globally. (@QuentinBisson)
 
+- Don't specify replica count for StatefulSets when autoscaling is enabled (@captncraig)
```

Comment:
Shouldn't this also be done for deployments?

Reply:
Thats where I got the snippet from :)